impeller/geometry/path.h

```diff
@@ -90,10 +90,29 @@ class Path {
 
   /// One or more contours represented as a series of points and indices in
   /// the point vector representing the start of a new contour.
+  ///
+  /// Polylines are ephemeral and meant to be used by the tessellator. They do
+  /// not allocate their own point vectors to allow for optimizations around
+  /// allocation and reuse of arenas.
   struct Polyline {
+    /// The signature of a method called when it is safe to reclaim the point
+    /// buffer provided to the constructor of this object.
+    using PointBufferPointer = std::unique_ptr<std::vector<Point>>;
+    using ReclaimPointBuffer = std::function<void(PointBufferPointer)>;
```

Comment:
I feel like these `using` statements make the code harder to follow.

Reply:
I was getting tired of typing out `std::unique_ptr<std::vector<Point>>` in a large number of places.

Reply:
If you want to keep these I think you show follow the conventions (inconsistently) used elsewhere in the codebase, the "Ptr" and "Callback" suffixes.

Reply:
Done